examples/chat-13B.sh

```diff
@@ -1,4 +1,18 @@
-#!/bin/bash
+#!/usr/env/bin bash
+
+while getopts "m:u:a:t:p:g:" opt; do
+  case $opt in
+    m) MODEL="--model $OPTARG";;
```

Comment:
Any reason for not doing it like this for consistency:

```suggestion
    m) MODEL="$OPTARG";;
```